### ISSUE TITLE
useradd fails in some cases

### ISSUE THREAD

**Issue by udeved** [artix-linux/opensysusers#1]
Hi Chris,

I tried to track down some issue with sysusers.
I added an updated hook, and this is output with some echo vars.

```
:: Starte post-transaction hooks...
(1/1) Updating system user accounts...
u
dbus
81

/
g
locate
21

/
u
mysql
89

/var/lib/mysql
u
ldap
439

Server" /var/lib/openldap
useradd: Ungültiges Home-Verzeichnis »Server" /var/lib/openldap«
g
rfkill
-

/
u
uuidd
68

/
g
vboxusers
108

/
```

The thing fails with openldap, the description in the ldap.conf, the gecko var that is I think gets a split string passed. Something with the parser is not right if the gecko has white spaces, ie if its not empty or not just a single word.

ldap.conf...

`u ldap 439 "LDAP Server" /var/lib/openldap`

**Comment by cromerc** [comment]
I see what the issue is, I will take a look at it.

**Comment by udeved** [comment]
I think 0.2.1 still has issues.
If I install openldap, a ldap user should be created.

(1/1) Creating temporary files...
chown: ungültiger Benutzer: „ldap“

but tmpfiles fails.

**Comment by cromerc** [comment]
tmpfiles is another software, I didn't write it, that is from upstream at OpenRC

I will double check with ldap, I tried with a some other conf files that had spaces in their gecos and it worked, just not openldap specificlaly

**Comment by cromerc** [comment]
Should now be fixed via 15ff42b18ab0c3cff84bbc5b65b431c1971ffa9c
